main-command/src/main/scala/sbt/Watched.scala

```diff
@@ -103,8 +112,127 @@ object Watched {
   val PollDelay: FiniteDuration = 500.milliseconds
   val AntiEntropy: FiniteDuration = 40.milliseconds
   def isEnter(key: Int): Boolean = key == 10 || key == 13
-  def printIfDefined(msg: String) = if (!msg.isEmpty) System.out.println(msg)
+  def printIfDefined(msg: String): Unit = if (!msg.isEmpty) System.out.println(msg)
+
+  type Task = () => State
+  type Setup = (State, Watched, String) => (State, Logger, Task => State)
```

Comment:
I tend to use the term "configuration" for these types of things.
Or maybe `WatchSetup`.

Reply:
I'll come up with an alternate name. I didn't put a lot of thought into this to be honest.